.gitignore

```diff
@@ -5,3 +5,6 @@
 .DS_Store
 venv/
 .pytest_cache/
+main.cpython-310.pyc
+__init__.cpython-310
+test_main.cpython-310-pytest-7.1.3
```

Comment:
It's not enough only to add these files to `.gitignore`, also you should remove them from your repo.

https://stackoverflow.com/questions/71922766/removing-pycache-from-git-repository